verbit/streaming_client.py

```diff
@@ -204,6 +225,47 @@ def set_logger(self, logger: logging.Logger = None):
     # ======== #
     # Internal #
     # ======== #
+    def _start_stream(self,
+                      media_generator: typing.Union[typing.Iterator[bytes], None] = None,
+                      media_config: typing.Union[MediaConfig, None] = None,
+                      response_types: ResponseType = ResponseType.Transcript) -> typing.Iterator[typing.Dict]:
```

Comment:
Consider renaming this to `_start_connection()`, which is both not imply there's an active `stream` is case it's `None`, and for lexical independent from the public `start_stream()` method so that it won't feel as-if it has to be renamed.

Reply:
See my comment suggesting to rename this to `_connect_and_start` - I think it's more descriptive, since this function both connects to the WebSocket, and starts:
- Media sender, if needed
- Response generator

Reply:
Renamed to `_connect_and_start`